templates/common/infra/bicep/core/host/aks-automatic-cluster.bicep

```diff
@@ -0,0 +1,100 @@
+metadata description = 'Creates an Azure Kubernetes Service (AKS) cluster with a system agent pool.'
+@description('The name for the AKS managed cluster')
+param name string
+
+@description('The name of the resource group for the managed resources of the AKS cluster')
+param nodeResourceGroupName string
+
+@description('The Azure region/location for the AKS resources')
+param location string = resourceGroup().location
+
+@description('Custom tags to apply to the AKS resources')
+param tags object = {}
+
+@description('Kubernetes Version')
+param kubernetesVersion string
```

Comment:
Just wondering - is there a reason we don't provide a default here like `aks-managed-cluster` does?